File: PYTHON/pauleigon/pauleigon.py
# Rating: ~ 1.6 / 10
# Link: https://open.kattis.com/problems/pauleigon

def main():
  list = input().split()
  if ((int(list[1]) + int(list[2]))//int(list[0]))%2 ==0:
	  print("paul")
  else:
	  print("opponent")


if __name__ == "__main__":
  main()

File: C++/pauleigon/pauleigon.cpp
/*
Rating: ~ 1.6 / 10
Link: https://open.kattis.com/problems/pauleigon
*/

#include<iostream>

using namespace std;

int main() {
	int serves;
	int score1;
	int score2;

	cin >> serves >> score1 >> score2;
	if ((score1+score2)/serves%2 == 0) {
		cout << "paul" << endl;
	}
	else {
		cout << "opponent" << endl;
	}
}
